tests/auth_tests/test_models.py

```diff
@@ -189,6 +191,17 @@ def forwards(apps, schema_editor):
         user = User.objects.get(username='user1')
         self.assertTrue(user.check_password('secure'))
 
+    def test_normalize_email_deprecation(self):
+        msg = 'BaseUserManager.normalize_email() is deprecated in favor of '
+        msg += 'AbstractBaseUser.normalize_email() and will be removed in Django 4.1.'
```

Comment:
```suggestion
        msg = (
            'BaseUserManager.normalize_email() is deprecated in favor of '
            'AbstractBaseUser.normalize_email() and will be removed in Django 4.1.'
        )
```
I think the other examples in the code base are written like this.